src/Menus/HasMenuIcons.php

```diff
@@ -55,43 +78,55 @@ public function setIconUrl(string $url)
     }
 
     /**
-     * Returns the full icon tag: either a <i> tag for FontAwesome
-     * icons, or an <img> tag for images.
+     * Returns HTML tag depends Menu Icon type `iconUrl` or `faIcon`
+     *
+     * Returns the full icon tag: either a `<i>` tag for FontAwesome icons,
+     * or an `<img>` tag for images.
+     *
+     * @param string $classOrIconUrl FontAwesome Icon Class Name or Icon URL
+     *
+     * @return string
```

Comment:
There is no need to duplicate callouts in PHPDoc if they are set in the method. a note to all such changes

Reply:
Thanks for review @neznaika0 

This is done by a VSCode's plugin called [PHP Sniffer & Beautifier](https://marketplace.visualstudio.com/items?itemName=ValeryanM.vscode-phpsab). I will  remove those unnecessary comment and run `composer run style` first before commit. 

Please review it again and tell me if any updates needed.